### ISSUE TITLE
[WCAG] Keyboard Navigation Improvements

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/GeoView#3490]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

**1 - Chart Panel**
When the Chart panel is auto-opened (by pressing on an item within a map, for example), a close button is not displayed on the right side of the panel. Therefore, a keyboard user would not have a quick way of closing the right panel.

This should be updated to behave more like the details panel (specifics of the UX to be determined).

Note: The existing implementation requires pressing a LayerListItem (left panel) to activate the right panel focus trap / close button.

**2 - Footer Bar**
Pressing "esc" within footer bar panels does not bring focus to the corresponding (toggle) tab. Instead, the exit button first needs to be accessed. Ideally, "esc" would focus on the corresponding tab and the exit button could be removed.

**3 - Map Fullscreen - ESC Key Handling Conflict**
When in map fullscreen mode, the "esc" key closes fullscreen. It does not close panels. This is a reasonable behaviour.

**Comment by kenchase** [comment]
**1 - Chart Panel**
When a map interaction (such as clicking a feature) triggers multiple panels to auto-open simultaneously (e.g., Details panel in the appBar and Chart panel in the footerBar), only one panel can receive the focus trap at a time.

The Chart panel can't display the WCAG UI (focus trap + close button) until a layer within its panel has been pressed.

This limitation has been documented in the A11Y markdown doc.

(Chart Panel Example)[https://canadian-geospatial-platform.github.io/geoview/public/demos-navigator.html?config=./configs/navigator/demos/13-package-geochart.json]

**2 - Footer Bar**
Can't replicate this issue. This was likely fixed along the way while resolving other issues.

**3 - Map Fullscreen - ESC Key Handling Conflict**
When in map fullscreen mode, the "esc" key closes fullscreen. It does not close panels. This is a reasonable and expected behaviour. This is the convention because the Fullscreen API spec treats ESC as a user-safety mechanism, not a UI action apps get to prioritize.

A11Y markdown doc updated to reflect fullscreen/ESC key constraints.
